### PR TITLE
Actualized VPN table

### PR DESCRIPTION
**Pull request by c0rdis**
## Description
The VPN table was quite outdated - reviewed and fixed number of servers, price and in case of Perfect Privacy - country (from Panama to Switzerland, see https://www.perfect-privacy.com/imprint). Also smaller fixes for better sorting of the table.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -211,7 +211,7 @@ <h3 id="usa" class="anchor">Why is it not recommended to choose a US-based servi
         <tr>
           <td data-value="AirVPN">
             <a href="https://airvpn.org/"><img alt="AirVPN" src="assets/img/provider/AirVPN.png" width="200" height="70"></a></td>
-          <td data-value="70">54 €</td>
+          <td data-value="54">54 €</td>
```
Comment (Shifterovich) [inline index.html]: Here you use EUR in data-value

File: index.html [hunk 2]
```diff
@@ -255,9 +255,9 @@ <h3 id="usa" class="anchor">Why is it not recommended to choose a US-based servi
           <td data-value="ExpressVPN">
             <a href="https://www.expressvpn.com/"><img alt="ExpressVPN" src="assets/img/provider/ExpressVPN.png" width="200" height="70"></a>
           </td>
-          <td data-value="100">$ 99.95</td> <!-- USD on March 21, 2017 -->
+          <td data-value="99">$ 99.95</td> <!-- USD on December 28, 2018 -->
```
Comment (Shifterovich) [inline index.html]: And here you use USD in data-value.

File: index.html [hunk 3]
```diff
@@ -266,7 +266,7 @@ <h3 id="usa" class="anchor">Why is it not recommended to choose a US-based servi
           <td data-value="FrootVPN">
             <a href="https://www.frootvpn.com/"><img alt="FrootVPN" src="assets/img/provider/FrootVPN.png" width="200" height="70"></a>
           </td>
-          <td data-value="36">$ 35.88</td>
+          <td data-value="35">$ 35.88</td>
```
Comment (Shifterovich) [inline index.html]: This should be rounded to 36.

File: index.html [hunk 4]
```diff
@@ -277,9 +277,9 @@ <h3 id="usa" class="anchor">Why is it not recommended to choose a US-based servi
           <td data-value="hide.me">
             <a href="https://hide.me/"><img alt="hide.me" src="assets/img/provider/hide.me.png" width="200" height="70"></a>
           </td>
-          <td data-value="65">$ 65.04</td>
+          <td data-value="0">Free</td>
```
Comment (Shifterovich) [inline index.html]: Maybe this should mention that there is a 2 GB limit?

Comment (c0rdis) [reply index.html]: Not sure - if we do that, we should probably also mention limitations of other cheapest/free options - e.g. that of ProtonVPN

File: index.html (hunk 1 above)
Comment (c0rdis) [reply index.html]: True - although it's visually much easier to get the data sorted according to numbers. Also, we wouldn't be dependent on exchange rates (e.g. now 54 € ~ $62)

File: index.html (hunk 2 above)
Comment (c0rdis) [reply index.html]: See comment above

File: index.html (hunk 4 above)
Comment (Shifterovich) [reply index.html]: If it's possible to do that inline while keeping the table nice, sure. Otherwise, maybe add a \* footnote to these two?

**Comment by c0rdis** [comment]
Looks like I've got another commit as part of this PR - related to https://github.com/privacytoolsIO/privacytools.io/pull/621#issuecomment-450481258. Please review it as well.

**Comment by Shifterovich** [comment]
I'll extract it into a separate branch. One change per PR.

File: index.html (hunk 2 above)
Comment (c0rdis) [reply index.html]: For now, manually updated to reflect USD values, so resolving

File: index.html (hunk 1 above)
Comment (c0rdis) [reply index.html]: For now, manually updated to reflect USD values, so resolving

**Comment by c0rdis** [comment]
Sure, sorry

**Pull request merged**